dags/bqetl_serp.py

```diff
@@ -57,6 +57,9 @@
         ],
         date_partition_parameter=None,
         depends_on_past=False,
+        parameters=[
+            'submission_date:DATE:{{ (execution_date - macros.timedelta(hours=24)).strftime("%Y%m%d") }}'
+        ],
```

Comment:
So if this query runs for 2023-10-05, the query itself subtracts one day. Should the results be stored in the partition for 2023-10-05 or 2023-10-04?

The current change will have it in 2023-10-04.

Reply:
The results should be stored in 10-05-2023